tgui/packages/tgui/interfaces/PreferencesMenu/MainPage.tsx

```diff
@@ -162,6 +166,29 @@ const ChoicedSelection = (props: {
           </Stack>
         </Stack.Item>
 
+        {
+          // BUBBER EDIT ADDITION: Better prefs: Make supplemental features display fully under the header
+          catalog.supplemental_features && (
+            <Stack.Item>
+              <PreferenceList
+                act={act}
+                preferences={(() => {
+                  // Lazy hack fraud method
+                  const supplementalsRecord: Record<string, unknown> =
+                    new Object() as Record<string, unknown>;
+                  catalog.supplemental_features.forEach((value) => {
+                    supplementalsRecord[value] =
+                      data.character_preferences.supplemental_features[value];
+                  });
+                  return supplementalsRecord;
+                })()}
+                randomizations={new Object() as Record<string, RandomSetting>}
```

Comment:
Why give randomizations a empty object? If you want to not do any randomization deal with making it not a required variable